frontera/contrib/backends/redis_backend/__init__.py

```diff
@@ -345,12 +347,9 @@ def links_extracted(self, _, links):
             if link_fingerprint in links_deduped:
                 continue
             links_deduped[link_fingerprint] = link
-        self._redis_pipeline(
-            "write links extracted",
-            lambda pipe, links: map(
-                lambda (fingerprint, link): pipe.hmset(fingerprint, self._create_link_extracted(link)), links.items()),
-            links_deduped
-        )
+        pipeline = RedisPipeline(self._pool)
```

Comment:
is it possible to have a permanent instance of RedisPipeline/RedisOperation per RedisMetadata. There is overhead connected with instantiation of RedisPipeline->StricRedis, Logger